### PR TITLE
AIRNERIS-58/70-backoffice-&-category page

### PR DESCRIPTION
**Pull request by hallycsse**
back office categories :
image
back office products : 
image
category page : 
image

# All of the pages are only front, as seen with @KyDev25 api/filter will be done later.

### CODE REVIEW

**Comment by hallycsse** [comment]
And for the fonction length I want to discuss about it tomorrow in the sprint review and if its not changed I will update my work.

**Comment by cedricgautier** [comment]
Can you please separate these two tickets to have a separation of concerns. Thanks

Minimum two pull requests : 
1.  Back-Office
2. Category Page

**Comment by hallycsse** [comment]
separated as @cedricgautier wanted with his help. thx

File: front/src/components/BackOfficeModal.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { CheckCircleIcon, PencilIcon, TrashIcon, XCircleIcon } from "@heroicons/react/24/outline"
+import Link from "next/link"
+import { useState } from "react"
+
+// eslint-disable-next-line no-unused-vars
+const BackOfficeModal = ({ handleOpenModal, id }) => {
+  const [askDelete, setAskDelete] = useState(false)
+  const handleAskDelete = () => setAskDelete(!askDelete)
+  const handleDelete = () => {
+    // To do
```
Comment (avetisk) [inline front/src/components/BackOfficeModal.jsx:10]: 👀

File: front/src/components/BackOfficeNav.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <nav
+      className={`h-full w-fit md:left-0 md:w-2/5 lg:w-1/3 md:static bg-tertiary border-r-2 z-20 ${
+        isOpen ? "left-0 ease-in-out duration-300 absolute" : "-left-48  ease-in-out duration-300 absolute"
+      }`}
+    >
+      <BackOfficeNavPM page={page} />
+      <ul className="flex flex-col px-3 gap-6 pb-6 ">
+        <li className="md:text-xl border-b leading-extra-loose md:leading-extra-loose">User Management</li>
+        <li className={clsx("py-1.5", page === "users" ? "bg-black/30 rounded-sm" : "")}>
```
Comment (avetisk) [inline front/src/components/BackOfficeNav.jsx:20]: You can write it like:
```js
clsx("py-1.5", page === "users" && "bg...")```

File: front/src/pages/back-office/products.jsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  { id: 20, name: "CANAPÉ SAUFA", price: 2450, image: "https://picsum.photos/90/70" },
+  { id: 30, name: "CHAISE DES LANDES", price: 575, image: "https://picsum.photos/90/70" },
+  { id: 40, name: "CANAPÉ BLANC", price: 1200, image: "https://picsum.photos/90/70" },
+  { id: 50, name: "CANAPÉ SAUFA", price: 2450, image: "https://picsum.photos/90/70" },
+  { id: 60, name: "CHAISE DES LANDES", price: 575, image: "https://picsum.photos/90/70" },
+  { id: 14, name: "CANAPÉ BLANC", price: 1200, image: "https://picsum.photos/90/70" },
+  { id: 24, name: "CANAPÉ SAUFA", price: 2450, image: "https://picsum.photos/90/70" },
+  { id: 34, name: "CHAISE DES LANDES", price: 575, image: "https://picsum.photos/90/70" },
+  { id: 44, name: "CANAPÉ BLANC", price: 1200, image: "https://picsum.photos/90/70" },
+  { id: 54, name: "CANAPÉ SAUFA", price: 2450, image: "https://picsum.photos/90/70" },
```
Comment (avetisk) [inline front/src/pages/back-office/products.jsx:27]: Mock data shouldn't be here.

**Pull request closed without merging**